### PR TITLE
First review of header files

### PR DESCRIPTION
**Pull request by francoiseblanc**
This is the first review of the header files. I'll make another once we finish coding. The only thing I would like you to confirm is the  [in], [out], and [in,out] for your fonctions. Thanks!

### CODE REVIEW

**Review by staubibr — APPROVED** [review]
Looks good to me!

**Review by AidanFahlman — APPROVED** [review]
Looks good and makes sense to me, great job!

File: include/simulation_lifecycle/spatial_analysis.h [hunk 1]
```diff
@@ -1,50 +1,52 @@
...
 
 /**
- * Registers an operation to a global list of available operations to be used by the execute_workflow function
- * @param name a string containing the name of the operation to register
- * @param execute a function that executes the registered operation
- * @return nothing
+ * @brief Registers an operation to a global list of available operations to be used by the execute_workflow function.
+ * @param[in] name a string containing the name of the operation to register.
+ * @param[in] execute a function that executes the registered operation.
+ * @return nothing.
```
Comment (romancardenas) [inline include/simulation_lifecycle/spatial_analysis.h:35]: You can remove this line if the function returns void

File: include/simulation_lifecycle/spatial_analysis.h [hunk 2]
```diff
@@ -1,50 +1,52 @@
...
  */
 void register_operation(char * name, int (* execute)(char * id, node_t * data_sources, cJSON * parameters, node_t ** results));
 
 /**
- * Registers all available operations to a global list of available operations to be used by the execute_workflow function
- * @return nothing
+ * @brief Registers all available operations to a global list of available operations to be used by the
+ * execute_workflow function.
+ * @return nothing.
```
Comment (romancardenas) [inline include/simulation_lifecycle/spatial_analysis.h:42]: Same here.

**Review by romancardenas — APPROVED** [review]
Great job, @francoiseblanc ! I made a couple of silly comments. Can you address them?

**Comment by romancardenas** [comment]
Hey @francoiseblanc ! Note that I pushed a commit to your branch! It deals with Git conflicts. Please pull the changes to your local repository before addressing my suggestions

**Review by romancardenas — APPROVED** [review]

**Pull request merged**